schemas/telemetry/focus-event/focus-event.1.parquetmr.txt

```diff
@@ -39,6 +39,7 @@ message focus_event {
           }
         }
       }
+
```

Comment:
We discussed cleaning this up with `string(STRIP "${${VARNAME}}" ${VARNAME})` but I would rather have it addressed by #105